app/screens/channel/channel_post_list/index.ts

```diff
@@ -36,6 +37,7 @@ const enhanced = withObservables(['channelId'], ({database, channelId}: {channel
                 }
 
                 const {earliest, latest} = postsInChannel[0];
+                logDebug('using as posts in channels', {earliest, latest, now: Date.now()});
```

Comment:
This is for every channel and every time this observable runs, not sure if there is a better way.

Also, maybe the channel id to be able to identify it somehow in the logs while looking for the affected channel?

Do we care about how many chunks or posts in a chunk?